src/core/Akka.Tests/Actor/Stash/ActorWithBoundedStashSpec.cs

```diff
@@ -39,33 +39,37 @@ protected override void OnReceive(object message)
         private void AfterWorldBehavior(object message) => Stash.Stash();
     }
 
-    public class StashingActorWithOverflow : UntypedActor, IWithBoundedStash
+    public class StashingActorWithOverflow : UntypedActor, IWithStash
```

Comment:
Made this actor configurable, rather than hard-coded to a single bounded stash size.